model-client/src/jsTest/kotlin/org/modelix/model/client2/ClientJSTest.kt

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2023.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.modelix.model.client2
+
+import GeneratedConcept
+import org.modelix.kotlin.utils.UnstableModelixFeature
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+@OptIn(UnstableModelixFeature::class)
+class ClientJSTest {
+
+    @Test
+    fun canAddChildrenWithUnregisterConcept() {
```

Comment:
```suggestion
    fun canAddChildrenWithUnregisteredConcept() {
```

Is that the intended meaning of the test case?

Reply:
Done.

And yes, that was the intended meaning.